Genre content DB.sql

```diff
@@ -0,0 +1,77 @@
+(4, 'Cozy Grove', 'Cozy Games', 'PC, NINTENDO SWITCH', 11.39, 'Welcome to Cozy Grove, a game about camping on a haunted, ever-changing island. As a Spirit Scout, youll wander the islands forest each day, finding new hidden secrets and helping soothe the local ghosts. With a little time and a lot of crafting, youll bring color and joy back to Cozy Grove!'),
+(5, 'Hogwarts Legacy', 'RPG', 'PC, XBOX, PLAYSTATION, NINTENDO SWITCH', 49.99, 'Hogwarts Legacy is an immersive, open-world action RPG. Now you can take control of the action and be at the center of your own adventure in the wizarding world.'),
+(6, 'God of War', 'RPG', 'PLAYSTATION', 39.99, 'Against a backdrop of Norse Realms torn asunder by the fury of the Aesir, they’ve been trying their utmost to undo the end times. But despite their best efforts, Fimbulwinter presses onward. Witness the changing dynamic of the father-son relationship as they fight for survival. Atreus thirsts for knowledge to help him understand the prophecy of Loki, as Kratos struggles to break free of his past and be the father his son needs.'),
+(7, 'Lies of P', 'RPG', 'PLAYSTATION, PC, XBOX', 49.99, 'Lies of P is a thrilling soulslike that takes the story of Pinocchio, turns it on its head, and sets it against the darkly elegant backdrop of the Belle Epoque era.'), 
+(8, 'Marvels Spider Man 2', 'RPG', 'PLAYSTATION', 69.99, 'Peter Parker and Miles Morales return for an exciting new adventure in the critically acclaimed Marvel’s Spider-Man franchise. Swing, jump and utilize the new Web Wings to travel across Marvel’s New York, quickly switching between Peter Parker and Miles Morales to experience different stories and epic new powers, as the iconic villain Venom threatens to destroy their lives, their city and the ones they love.'), 
+(9, 'STARDEW VALLEY', 'SIMULATION', 'PC, NINTENDO SWITCH, XBOX, PLAYSTATION', 10.99, 'Youve inherited your grandfathers old farm plot in Stardew Valley. Armed with hand-me-down tools and a few coins, you set out to begin your new life. Can you learn to live off the land and turn these overgrown fields into a thriving home?'),
+(10, 'Two Point Hospital', 'SIMULATION', 'PC, NINTENDO SWITCH, XBOX, PLAYSTATION', 24.99, 'Design stunning hospitals, cure peculiar illnesses and manage troublesome staff as you spread your budding healthcare organisation across Two Point County.'),
+(11, 'GAME DEV TYCOON', 'SIMULATION', 'PC', 8.50, 'In Game Dev Tycoon you replay the history of the gaming industry by starting your own video game development company in the 80s. Create best selling games. Research new technologies and invent new game types. Become the leader of the market and gain worldwide fans.'),
+(12, 'NAHEULBEUKS DUNGEON MASTER', 'SIMULATION', 'PC', 20.99, 'A dungeon in danger ! Build, manage, and defend your tower in the satirical heroic fantasy universe of Dungeon of Naheulbeuk. From a shaky establishment to an infamous lair!');
+
```

Comment:
Looks really good I like the amount the options

Reply:
Totally understandable I just picked some that I have played myself and random ones off of steam seems as we only need them for examples. Ideally if we were to make the entire website functional we would have 1000s of genre's and titles coming from an public API and wouldn't need a database for that at all. I will make note of this to mention in the documentation so our marker is aware